content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -25,31 +25,36 @@ is a replacement for the `istio-init` container that performs the same
 networking functionality but without requiring Istio users to enable elevated
 Kubernetes RBAC permissions.
 
-The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
+The Istio CNI plugin identifies Istio user application pods with Istio sidecars requiring traffic redirection and
+performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
 setup phase, thereby removing the [requirement for the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/)
 for users deploying pods into the Istio mesh.  The Istio CNI plugin
 replaces the functionality provided by the `istio-init` container.
 
-## Prerequisites
+## Install CNI
 
-1.  Install Kubernetes with the container runtime supporting CNI and `kubelet` configured
-    with the main [CNI](https://github.com/containernetworking/cni) plugin enabled via `--network-plugin=cni`.
-    *  AWS EKS, Azure AKS, and IBM Cloud IKS clusters have this capability.
-    *  Google Cloud GKE clusters require that the
-       [network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) feature
-       is enabled to have Kubernetes configured with `network-plugin=cni`.
-    *  OpenShift has CNI enabled by default.
+### Prerequisites
 
-1.  Install Kubernetes with the [ServiceAccount admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) enabled.
-    *  The Kubernetes documentation highly recommends this for all Kubernetes installations
-       where `ServiceAccounts` are utilized.
+1. Install Kubernetes with the container runtime supporting CNI and `kubelet` configured
+  with the main [CNI](https://github.com/containernetworking/cni) plugin enabled via `--network-plugin=cni`.
+    * AWS EKS, Azure AKS, and IBM Cloud IKS clusters have this capability.
+    * Google Cloud GKE clusters has CNI enabled when any of the following features is enabled:
+       [network policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy),
+       [intranode visibility](https://cloud.google.com/kubernetes-engine/docs/how-to/intranode-visibility),
+       [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity),
+       [pod security policy](https://cloud.google.com/kubernetes-engine/docs/how-to/pod-security-policies#overview),
+       and [dataplane v2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2).
```

Comment:
nit:
```suggestion
       or [dataplane v2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2).
```

Or maybe and/or. Not sure its needed, the prior part does say "when any of"